build/openwisp_dashboard/module_settings.py

```diff
@@ -49,6 +49,7 @@
     'allauth.socialaccount.providers.google',
     # openwisp-radius
     'openwisp_radius',
+    'openwisp'
```

Comment:
can we find a different name for this? It may be misunderstood.
Something like `openwisp_periodic_tasks` could work.

Reply:
I was able to do away without mentioning the app by:
```
app = Celery('openwisp', include=['openwisp.tasks'])
```
in the celery.py

Hence, this line is removed.